<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>147</width>
      <height>592</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>22</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="networkButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="arpButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="anticrackButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fakeapButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="devicesButton">
       <property name="minimumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>145</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>631</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>801</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TitleBar">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="Titlebar">
       <property name="minimumSize">
        <size>
         <width>141</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>141</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/res/image/title.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="setButton">
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minButton">
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maxButton">
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="minimumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>29</width>
         <height>18</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_CheckLogin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>572</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
